app/models/key_stage.rb

```diff
@@ -44,6 +44,19 @@ def secondary?
     %w[3 4].include?(level)
   end
 
+  def page_url
+    base_url = case Rails.env
```

Comment:
I would move this to somewhere shared, as we have this same block of code in multiple places, which makes it more work if we ever change the URL.

Also for the development one, I would potentially pull the URL from the env file, as people can have different setups for local testing. The others can be hard coded, but as per comment above this should be in a shared function